rs/nns/governance/canister/canister.rs

```diff
@@ -245,6 +251,40 @@ fn schedule_prune_following(delay: Duration) {
     });
 }
 
+thread_local! {
+    static BACKFILL_VOTING_POWER_REFRESHED_TIMESTAMPS_CHECKPOINT: RefCell<Bound<NeuronIdProto>> =
+        const { RefCell::new(Bound::Unbounded) };
+}
+
+fn schedule_backfill_voting_power_refreshed_timestamps(delay: Duration) {
+    ic_cdk_timers::set_timer(delay, || {
+        let original_checkpoint =
+            BACKFILL_VOTING_POWER_REFRESHED_TIMESTAMPS_CHECKPOINT.with(|p| *p.borrow());
+
+        let carry_on = || {
+            call_context_instruction_counter()
+                < MAX_BACKFILL_VOTING_POWER_REFRESHED_TIMESTAMPS_INSTRUCTIONS
+        };
+        let new_checkpoint = governance_mut()
+            .backfill_some_voting_power_refreshed_timestamps(original_checkpoint, carry_on);
+
+        BACKFILL_VOTING_POWER_REFRESHED_TIMESTAMPS_CHECKPOINT.with(|p| {
+            let mut borrow = p.borrow_mut();
+            *borrow = new_checkpoint;
+        });
+
+        let is_done = new_checkpoint == Bound::Unbounded;
+        if is_done {
+            return;
+        }
+
+        // Otherwise, continue later.
+        schedule_backfill_voting_power_refreshed_timestamps(
+            BACKFILL_VOTING_POWER_REFRESHED_TIMESTAMPS_INTERVAL,
+        );
+    });
+}
```

Comment:
This is very similar to the function above. (In fact, I used copy n' paste.) Ideally, I would factor out the common bits. I did not bother with that here, because this code is temporary anyway (as indicated by the TODO comment I added earlier in this file).